src/aat/resources/features/F-050/S-298.td.json

```diff
@@ -0,0 +1,81 @@
+{
+  "_guid_": "S-298",
+  "_extends_": "Base_Expecting_422",
+  "title": "must not validate when field validation fails",
+    "specs": ["a detailed profile in CCD",
+    "uses a field that doesn’t exist",
+    "uses a value that exceeds the field max limit",
+    "has the 422 return code"
+  ],
+  "productName": "CCD Data Store",
+  "operationName": "validation of a set of fields as Case worker",
+  "method": "POST",
+  "uri": "/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/validate",
+  "request": {
+    "pathVariables": {
+      "uid": "[[DYNAMIC]]",
+      "jid" : "AUTOTEST1",
+      "ctid" : "AAT"
+    },
+    "body" : {
+      "data": {
+        "$TextField£": "Text",
+        "NumberField": null,
+        "YesOrNoField": "Yes",
+        "PhoneUKField": "012341234123",
+        "EmailField": "abc@gmail.com",
+        "MoneyGBPField": "100",
+        "DateField": null,
+        "DateTimeField": "",
+        "TextAreaField": null,
+        "FixedListField": null,
+        "FixedRadioListField": null,
+        "MultiSelectListField": [],
+        "ComplexField": {
+          "ComplexTextField": null,
+          "ComplexFixedListField": null,
+          "ComplexNestedField": {
+            "NestedNumberField": null,
+            "NestedCollectionTextField": []
+          }
+        },
+        "CollectionField": [],
+        "AddressUKField": {
+          "AddressLine1": null,
+          "AddressLine2": null,
+          "AddressLine3": null,
+          "PostTown": null,
+          "County": null,
+          "PostCode": null,
+          "Country": null
+        }
+      },
+      "event": {
+        "id": "CREATE",
+        "summary": "",
+        "description": ""
```

Comment:
the content of the request body can be simplified by data reuse. you shouldn't repeat this body content in all test data. You can define it in a file and reuse it from there.

Reply:
Implemented. Created a request file : F-050_Test_Data_Base.td.json